RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java

```diff
@@ -418,10 +418,31 @@ private RegOutputFiles ripRegistryFile(String regFilePath, String outFilePathBas
             String errFilePath = outFilePathBase + "-full.err.txt"; //NON-NLS
             logger.log(Level.INFO, "Writing Full RegRipper results to: {0}", regOutputFiles.fullPlugins); //NON-NLS
             executeRegRipper(rrFullCmd, rrFullHome, regFilePath, fullType, regOutputFiles.fullPlugins, errFilePath);
+            try {
+                scanErrorLogs(errFilePath);
+            } catch (IOException ex) {
+            logger.log(Level.SEVERE, "Unable to run RegRipper", ex); //NON-NLS
+            this.addErrorMessage(NbBundle.getMessage(this.getClass(), "ExtractRegistry.execRegRip.errMsg.failedAnalyzeRegFile", this.getName()));
+        } 
         }
         return regOutputFiles;
     }
 
+    private void scanErrorLogs(String errFilePath) throws IOException {
+        File regfile = new File(errFilePath);
+        try (BufferedReader reader = new BufferedReader(new FileReader(regfile))) {
+            String line = reader.readLine();
+            while (line != null) {
+                line = line.trim();
+                if (line.contains("Error") || line.contains("@INC")) {
+                   logger.log(Level.WARNING, "Regripper file {0} contains errors from run", errFilePath); //NON-NLS
```

Comment:
Would it be worth also sending a notification to the user to let them know? Or is the fact that we are running regripper not explicitly stated in the Recent Activity module description?

Reply:
I just put the warning in the log for debugging purposes if needed. Since not all plugins are parsed in Autopsy this will log the plugins that actually get an error that autopsy may not use.

Reply:
Okay, last thing, could we make the string checking case insensitive?

Reply:
changed to check to for case insensitive, good catch.